### PR TITLE
feat: about page skeleton

### PR DESCRIPTION
**Pull request by jessicamcinchak**
Screenshot from 2023-01-13 11-03-39

### CODE REVIEW

File: pages/about.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+const AboutPage = () => {
+  return (
+    <div className="bg-black z-20 text-white max-w-full fixed inset-0 overflow-y-auto overflow-x-auto">
+      <nav className="flex justify-between ml-8 mr-8 mt-3 h-12">
+        <a href="/">Atlas of Ownership</a>
+        <a href="/" className="text-lg">X</a>
+      </nav>
```
Comment (jessicamcinchak) [inline pages/about.tsx:7]: we'll eventually want to move this nav bar out i suspect and check if next has an internal link element, but just hardcoded here for starters

File: pages/about.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+          </p>
+          <p className="mb-3">
+            Why? Because redesigning how we own land and property is the single most important reform project of our century. There is no path to a successful, flourishing future for humanity that does not include it.
+          </p>
+          <p className="mb-3">
+            The solutions that we need to build that future are already here, we just need to find them.
+          </p>
+          <blockquote className="mb-3 text-gray-400">
+            “The ultimate, hidden truth of the world is that it is something that we make, and could just as easily make differently” – David Graeber
+          </blockquote>
```
Comment (jessicamcinchak) [inline pages/about.tsx:40]: maybe it worth adding About page content to Sanity and querying in future? not sure how often we expect this to change, again just a simple hardcode for starters

Comment (DafyddLlyr) [reply pages/about.tsx:42]: This thought has also occurred to me on the explore patterns page where there's some hardcoded text - hardcoding for now makes sense I think.

File: pages/about.tsx [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import Link from "next/link";
+
+const AboutPage = () => {
+  return (
+    <div className="bg-black z-20 text-white max-w-full fixed inset-0 overflow-y-auto overflow-x-auto">
+      <nav className="flex justify-between ml-8 mr-8 mt-3 h-12">
+        <Link href="/">Atlas of Ownership</Link>
+        <Link href="/" className="text-lg">X</Link>
```
Comment (DafyddLlyr) [inline pages/about.tsx:8]: nit: We use the `<Close size={32} className="cursor-pointer" />` Carbon Icon for this on the entry page

**Review by DafyddLlyr — APPROVED** [review]

**Pull request merged**